index.html

```diff
@@ -58,5 +64,185 @@ <h3 class="dev-intro">I'am a Software Developer</h3>
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="tech-list">
+            <li>Html</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button type="button" class="btn1">See Project</button>
+        </div>
+
+        <div class="card-section">
+          <h2 class="Profesional">Profesional Art Printing Data</h2>
+          <p class="paragraph">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="tech-list">
+            <li>Html</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button type="button" class="btn1">See Project</button>
+        </div>
+
+        <div class="card-section">
+          <h2 class="Profesional">Profesional Art Printing Data</h2>
+          <p class="paragraph">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="tech-list">
+            <li>Html</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button type="button" class="btn1">See Project</button>
+        </div>
+
+        <div class="card-section">
+          <h2 class="Profesional">Profesional Art Printing Data</h2>
+          <p class="paragraph">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="tech-list">
+            <li>Html</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button type="button" class="btn1">See Project</button>
+        </div>
+
+        <div class="card-section">
+          <h2 class="Profesional">Profesional Art Printing Data</h2>
+          <p class="paragraph">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="tech-list">
+            <li>Html</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button type="button" class="btn1">See Project</button>
+        </div>
+      </section>
+    </section>
+
+    <section class="about-me">
+      <div class="about-me-content">
+        <h2 class="logo-message">About me</h2>
+        <p class="background">
+          Hello I am a software developer! I can help you build a product,
+          feature or websiteLook through some of my work and experience! If you
+          like what you see and have a project you need coded, do not hestiate
+          to contact me.
+        </p>
+      </div>
+
+      <button type="button" class="btn">Get My Resume</button>
```

Comment:
- [ ] Please the button "Get My Resume" follows the button styling: Active, Hover and Pressed
- [ ] see design Link: [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)
![ok](https://user-images.githubusercontent.com/63972670/164675918-d3944fc6-990e-4019-baab-a85385667ad9.PNG)

Reply:
Hi @[OLIPLICHE](https://github.com/OLIPLICHE) OLIPLICHE...
my buttons are working when I don't  inspect on the browser, when I inspect the hover effect is inactive, I have borrowed fro figma and changed some styling.